### PR TITLE
Add kino shorts

### LINKED ISSUE #256: Add Kino.Shortcuts (or Kino.Shorts if we want to be cheeky)
A module we can import with convenience functions to building text, markdown, mermaid, image, tree, frame, etc. This is so we can write:

    text("Hello")

Instead of:

    Kino.Text.new("hello")

### PR DESCRIPTION
**Pull request by josevalim**
Closes #256

### CODE REVIEW

**Comment by josevalim** [comment]
Some notes:

1. I did not include `Kino.Process` as it is too specific

2. I also did not include `Kino.Control`. It would be ok to use them with `Kino.listen` but `Kino.Control.form` would still require `Kino.Input`, so I kept them as is

**Comment by josevalim** [comment]
Some basic smoke tests are missing but this should be the bulk of the API.

File: lib/kino/shorts.ex [hunk 1]
```diff
@@ -0,0 +1,573 @@
...
+
+      import Kino.Shorts
+      text("Hello!")
+
+  """
+  @spec text(String.t()) :: Kino.Text.t()
+  def text(text), do: Kino.Text.new(text)
+
+  @doc ~S'''
+  `Kino.Mermaid` renders Mermaid graphs:
```
Comment (jonatanklosko) [inline lib/kino/shorts.ex:135]:
```suggestion
  Renders Mermaid graphs:
```

File: lib/kino/shorts.ex [hunk 2]
```diff
@@ -0,0 +1,573 @@
...
+
+      html("""
+      <h3>Look!</h3>
+
+      <p>I wrote this HTML from <strong>Kino</strong>!</p>
+      """)
+
+  '''
+  @spec html(String.t()) :: Kino.HTML.t()
+  def html(html), do: Kino.HTML.new(html)
```
Comment (jonatanklosko) [inline lib/kino/shorts.ex:205]: Shouldn't all of them be `defdelegate`?

Comment (josevalim) [reply lib/kino/shorts.ex:205]: the delegate version with `:as` ends up being more verbose, not worth it IMO.

File: lib/kino/shorts.ex [hunk 3]
```diff
@@ -0,0 +1,573 @@
...
+      * `:pcm_f32` (default) - the PCM (32-bit float) format. Note that
+        the binary uses native system endianness. Such binary can be
+        directly converted to an `Nx` tensor, with no additional decoding
+
+      * `:wav`
+
+    * `:sampling_rate` - the sampling rate (samples per second) of
+      the audio data. Defaults to `48_000`
+
+  """
```
Comment (jonatanklosko) [inline lib/kino/shorts.ex:522]: I'm concerned with duplicating all of the docs and options and keeping them in sync. Could we just point users towards the original docs?

File: lib/kino/shorts.ex (hunk 2 above)
Comment (jonatanklosko) [reply lib/kino/shorts.ex:205]: Ah it's the same number of dispatches, nevermind!

File: lib/kino/shorts.ex (hunk 3 above)
Comment (josevalim) [reply lib/kino/shorts.ex:522]: Yeah, I am torn. We want to make it accessible for terminal and autocompletion but in this case I think it is worth just pointing to the original docs. I will update later.

**Review by jonatanklosko — APPROVED** [review]
🚢

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**